Dockerfile-alpine.template

```diff
@@ -2,20 +2,19 @@ FROM alpine:0.0
 
 ENV NODE_VERSION 0.0.0
 
-RUN addgroup -g 1000 node \
-    && adduser -u 1000 -G node -s /bin/sh -D node \
-    && apk add --no-cache \
-        libstdc++ \
-    && apk add --no-cache --virtual .build-deps \
-        binutils-gold \
-        curl \
-        g++ \
-        gcc \
-        gnupg \
-        libgcc \
-        linux-headers \
-        make \
-        python \
+RUN ARCH= && alpineArch="$(arch)" \
+  && case "${alpineArch##*-}" in \
```

Comment:
Same as previous comment:

Is this needed? x64 is the only option so this can be simplified be removing the alpine specific case statement